### PR TITLE
For review

### PR DESCRIPTION
**Pull request by mkclcr**
For code review.

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -19,7 +19,6 @@
 <script type="text/javascript" src="src/js/bootstrap.min.js"></script>
```
Comment (mkclcr) [inline app/index.html]: Remove all duplicate links.

File: app/index.html [hunk 2]
```diff
@@ -19,7 +19,6 @@
...
 <script type="text/javascript" src="src/js/bootstrap.min.js"></script>
```
Comment (mkclcr) [inline app/index.html]: Remove duplicates 1

File: app/index.html [hunk 3]
```diff
@@ -19,7 +19,6 @@
...
 <script type="text/javascript" src="src/js/bootstrap.min.js"></script>
```
Comment (mkclcr) [inline app/index.html]: Remove duplicates 2

File: app/index.html (hunk 1 above)
Comment (mahendra2015) [inline app/index.html]: done.

File: app/index.html (hunk 2 above)
Comment (mahendra2015) [inline app/index.html]: done.

File: app/index.html (hunk 3 above)
Comment (mahendra2015) [inline app/index.html]: done.

**Pull request merged**